### PR TITLE
Experiments.html page for our CDN

### PR DESCRIPTION
**Pull request by dvoytenko**
Partial for #897.

/cc @cramforce

### CODE REVIEW

**Comment by cramforce** [comment]
Can the gulp code be de-duplicated?

Please add text to the README and some warning to the experiments page.

**Comment by dvoytenko** [comment]
@cramforce Actually, I wanted to keep get/set cookie separate - there are very different cases where one need to read and write.

**Comment by dvoytenko** [comment]
@cramforce I think I actually misunderstood. Yes, we discussed this with @erwinmombay today. I think we'll be doing a bit of a refactoring on gulpfile soon.

File: src/cookies.js [hunk 1]
```diff
@@ -44,3 +44,19 @@ export function getCookie(win, name) {
...
+/**
+ * Sets the value of the cookie. The cookie access is restricted and must
+ * go through the privacy review. Before using this method please file a
+ * GitHub issue with "Privacy Review" label.
+ *
+ * @param {!Window} win
+ * @param {string} name
+ * @param {string} value
+ * @param {time} expritationTime
+ * @return {?string}
```
Comment (erwinmombay) [inline src/cookies.js]: dont see a return

File: src/cookies.js [hunk 2]
```diff
@@ -44,3 +44,19 @@ export function getCookie(win, name) {
...
+
+/**
+ * Sets the value of the cookie. The cookie access is restricted and must
+ * go through the privacy review. Before using this method please file a
+ * GitHub issue with "Privacy Review" label.
+ *
+ * @param {!Window} win
+ * @param {string} name
+ * @param {string} value
+ * @param {time} expritationTime
```
Comment (erwinmombay) [inline src/cookies.js]: expirationTime(?) (same for actual parameter)

**Comment by erwinmombay** [comment]
yeah i am ok with this for now, We'll still doing the refactor into individual task files (working on this).

minor nit comments, otherwise. LGTM

File: src/cookies.js (hunk 1 above)
Comment (dvoytenko) [inline src/cookies.js]: Done

File: src/cookies.js (hunk 2 above)
Comment (dvoytenko) [inline src/cookies.js]: Done

**Pull request merged**